### PR TITLE
POC: priority overflow with wrapping

### PR DESCRIPTION
**Pull request by miroslavstastny**
POC

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Avatar-mount_base.html">650</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Avatar-mount.html">632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Button-mount_base.html">301</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Button-mount.html">302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Field-mount_base.html">1149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Field-mount.html">1133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProvider-mount_base.html">695</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProvider-mount.html">725</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-mount_base.html">84</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-mount.html">83</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">34</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">36</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">74</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">74</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/MakeStyles-mount_base.html">866</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/MakeStyles-mount.html">853</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Persona-mount_base.html">1748</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/Persona-mount.html">1720</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/SpinButton-mount_base.html">1433</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/SpinButton-mount.html">1391</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/SwatchPicker-mount_base.html">1629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/32909/perf-test-react-components/SwatchPicker-mount.html">1658</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-overflow/library/src/components/Overflow.tsx [hunk 1]
```diff
@@ -20,6 +20,7 @@ export type OverflowProps = Partial<
   Pick<ObserveOptions, 'overflowAxis' | 'overflowDirection' | 'padding' | 'minimumVisible'>
```
Comment (fabricteam) [inline packages/react-components/react-overflow/library/src/components/Overflow.tsx:20]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32909&commitId=aba8f117a9382fb96120ef328b04c2c724a58eaf&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|2| Changed|
|Avatar Converged.badgeMask.chromium.png|7| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1166| Changed|
|Drawer.Full Overlay Dark Mode.chromium.png|995| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2113| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

**Comment by miroslavstastny** [comment]
Closing the POC, FUI changes are in separate PR #33031

**Pull request closed without merging**